# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/gc/Tracer.cpp
# Commit: e9c2f256240a
# Full Hash: e9c2f256240a1f9bb09b10ac641bbd141eedb809
# Author: Jan de Mooij <jdemooij@mozilla.com>
# Date: 2021-03-06 09:33:16
# Regressor Bug: 1689413
# File Overlap Count: 1
# Description:
#   Bug 1689413 part 2 - Remove owned BaseShapes. r=jonco
#   
#   At this point all BaseShapes are unowned (= immutable, shared) so we can remove the
#   UnownedBaseShape type and the `BaseShape::unowned_` field.
#   
# ==============================================================================

diff -r 1ae4c30bb931 -r e9c2f256240a js/src/gc/Tracer.cpp
--- a/js/src/gc/Tracer.cpp	Sat Mar 06 01:05:19 2021 +0000
+++ b/js/src/gc/Tracer.cpp	Sat Mar 06 01:05:19 2021 +0000
@@ -97,7 +97,6 @@
 void gc::TraceCycleCollectorChildren(JS::CallbackTracer* trc, Shape* shape) {
   do {
     MOZ_ASSERT(shape->base());
-    shape->base()->assertConsistency();
 
     // Don't trace the propid because the CC doesn't care about jsid.
 